argparse/argparse.py

```diff
@@ -3,7 +3,10 @@
 """
 
 import sys
-from _collections import namedtuple
+try:
+    from _collections import namedtuple
+except ImportError:
+    from collections import namedtuple
```

Comment:
Most of the other modules in micropython-lib won't run under CPython (eg due to ffi module).

Don't you anyway want to compare with CPython's argparse, not this one?  Eg:
```
$ mkdir cpy
$ cp test_argparse.py cpy
$ cd cpy
$ python3 test_argparse.py
```
(This way CPython will import its own argparse, not ours.)